### PR TITLE
BC-5736 - Redis for tldraw

### PR DESCRIPTION
**Pull request by davwas**
# Description

## Links to Tickets or other pull requests
https://ticketsystem.dbildungscloud.de/browse/BC-5736
https://github.com/hpi-schul-cloud/dof_app_deploy/pull/701
https://github.com/hpi-schul-cloud/schulcloud-server/pull/4542

## Approval for review

- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

**Review by VikDavydiuk — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -52,27 +25,35 @@
...
+		"@testing-library/user-event": "^14.5.1",
 		"@types/jest": "^29.5.2",
-		"@types/mongodb": "^4.0.7",
-		"eslint-config-prettier": "^8.8.0",
-		"eslint-plugin-no-only-tests": "^3.1.0",
-		"eslint-plugin-prettier": "^5.0.0",
-		"eslint-plugin-promise": "^6.1.1",
-		"jest": "^29.6.0",
+		"@types/node": "^20.10.3",
+		"@types/react": "^18.2.42",
```
Comment (CeEv) [inline package.json:48]: You can go to 18.2.45

File: src/App.tsx [hunk 2]
```diff
@@ -2,6 +2,7 @@ import { useEffect } from 'react';
 import { Tldraw, useFileSystem } from '@tldraw/tldraw';
 import { useUsers } from 'y-presence';
 import { useCookies } from 'react-cookie';
+
```
Comment (CeEv) [inline src/App.tsx:5]: The correct order we want in all projects between relative pathes and simlinks+packages. Also sorting. But the most do not want the empty line between the imports. If the IDE make the sorting than this line is removed and produce a change line.

File: src/hooks/useMultiplayerState.ts [hunk 3]
```diff
@@ -24,12 +25,12 @@ export const room = new Room<TldrawPresence>(awareness, {});
 
 const STORAGE_SETTINGS_KEY = 'sc_tldraw_settings';
 
-const getUserSettings = (): TDSnapshot['settings'] | undefined => {
+export const getUserSettings = (): TDSnapshot['settings'] | undefined => {
```
Comment (CeEv) [inline src/hooks/useMultiplayerState.ts:28]: You export it, but i do not see on which place it is importat in this PR.

**Review by CeEv — CHANGES_REQUESTED** [review]

**Review by CeEv — DISMISSED** [review]
What has this pr to do with redis?

File: package.json (hunk 1 above)
Comment (davwas) [reply package.json]: done

File: src/App.tsx (hunk 2 above)
Comment (davwas) [reply src/App.tsx]: fixed

**Pull request closed without merging**